scripts/run_e2e_tests.sh

```diff
@@ -25,7 +25,7 @@ if [ -z "$FRONTEND_FOLDER" ]; then
 else
   cd $FRONTEND_FOLDER
 fi
-npm run start:ci &
+npm run start:e2e &
```

Comment:
I don think `start:e2e` is any different. What about changing the line in https://github.com/opencollective/opencollective-frontend/blob/42054c2bfee6c9eeac50a77b4d018cd33a988df4/package.json#L187 to:

```diff
- "start:ci": "nyc --silent --exclude \".next/**\" node server",
+ "start:ci": "nyc --silent --exclude \".next/**\" node dist/server",
```